adafruit_bno055.py

```diff
@@ -143,44 +143,110 @@ def _reset(self):
-        +------------------+-------+---------+------+----------+
-        | COMPASS_MODE     |   X   |   X     |  -   |     X    |
-        +------------------+-------+---------+------+----------+
-        | M4G_MODE         |   X   |   X     |  -   |     -    |
-        +------------------+-------+---------+------+----------+
-        | NDOF_FMC_OFF_MODE|   X   |   X     |  X   |     X    |
-        +------------------+-------+---------+------+----------+
-        | NDOF_MODE        |   X   |   X     |  X   |     X    |
-        +------------------+-------+---------+------+----------+
+        .. code-block::
+
+            legend: x=on, -=off
+            +------------------+-------+---------+------+----------+
+            | Mode             | Accel | Compass | Gyro | Absolute |
+            +==================+=======+=========+======+==========+
+            | CONFIG_MODE      |   -   |   -     |  -   |     -    |
+            +------------------+-------+---------+------+----------+
+            | ACCONLY_MODE     |   X   |   -     |  -   |     -    |
+            +------------------+-------+---------+------+----------+
+            | MAGONLY_MODE     |   -   |   X     |  -   |     -    |
+            +------------------+-------+---------+------+----------+
+            | GYRONLY_MODE     |   -   |   -     |  X   |     -    |
+            +------------------+-------+---------+------+----------+
+            | ACCMAG_MODE      |   X   |   X     |  -   |     -    |
+            +------------------+-------+---------+------+----------+
+            | ACCGYRO_MODE     |   X   |   -     |  X   |     -    |
+            +------------------+-------+---------+------+----------+
+            | MAGGYRO_MODE     |   -   |   X     |  X   |     -    |
+            +------------------+-------+---------+------+----------+
+            | AMG_MODE         |   X   |   X     |  X   |     -    |
+            +------------------+-------+---------+------+----------+
+            | IMUPLUS_MODE     |   X   |   -     |  X   |     -    |
+            +------------------+-------+---------+------+----------+
+            | COMPASS_MODE     |   X   |   X     |  -   |     X    |
+            +------------------+-------+---------+------+----------+
+            | M4G_MODE         |   X   |   X     |  -   |     -    |
+            +------------------+-------+---------+------+----------+
+            | NDOF_FMC_OFF_MODE|   X   |   X     |  X   |     X    |
+            +------------------+-------+---------+------+----------+
+            | NDOF_MODE        |   X   |   X     |  X   |     X    |
+            +------------------+-------+---------+------+----------+
 
         The default mode is ``NDOF_MODE``.
+
+        | You can set the mode using the line below:
+        | ``sensor.mode = adafruit_bno055.ACCONLY_MODE``
+        | replacing ``ACCONLY_MODE`` with the mode you want to use
+
+        .. data:: CONFIG_MODE
```

Comment:
Does this render it on the module or class?

Reply:
Class. Here's what it looks like. I can have it render on the module, so it starts right after the author, but I thought this way was a bit cleaner.

![docs](https://user-images.githubusercontent.com/33632497/86293125-0af27c00-bbc0-11ea-8b66-238a109b41f3.png)